src/App.js

```diff
@@ -1,25 +1,104 @@
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+import { Component } from 'react';
+import axios from 'axios';
+import Header from './Componenents/Header.js'
+import Footer from './Componenents/Footer.js'
+import 'bootstrap/dist/css/bootstrap.min.css';
+import Container from 'react-bootstrap/Container'
+import Form from 'react-bootstrap/Form'
+import Button from 'react-bootstrap/Button'
+import Row from 'react-bootstrap/Row'
+import Col from 'react-bootstrap/Col'
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchQuery: '',
+      location: {},
+      error: false,
+    }
+  }
+
+  getLocation = async (event) => {
+    event.preventDefault();
+
+    const url = `https://us1.locationiq.com/v1/search.php?key=${process.env.REACT_APP_CITY_KEY}&q=${this.state.searchQuery}&format=json`;
+
+    try {
+      const response = await axios.get(url);
+      console.log(response)
+      const location = response.data[0];
+
+      this.setState({
+        location,
+        error: false,
+      });
+
+    } catch (error) {
+      console.error('Unable to find city', this.state.searchQuery);
+      this.setState({ error: true });
+    }
+  }
+
+  render() {
+    return (
+
+
+      <>
+        <Header />
+        <Container fluid>
+          <Form>
+            <Row className="justify-content-md-center">
+              <Col sm={3} className="my-1">
+                <Form.Label htmlFor="inlineFormInputName" visuallyHidden>
+                  Please Enter City Name
+                </Form.Label>
+                <Form.Control onChange={(event) => this.setState({ searchQuery: event.target.value })} placeholder="Ex: Seattle" />
```

Comment:
Nice work on the lab so far!

Feedback here: Please remove setState call here as it can cause funky behavior (infinite loops and makes component less maintainable). Instead create a method to handle event outside of render and then pass it to onChange. 

(Sara and JB spoke about why setState in renders are not usually a good idea) 👍  Grab a TA if you need any additional help. Again, nice work!